src/client/routes/Portfolio/index.tsx

```diff
@@ -20,27 +21,20 @@ import {
   DetailCard,
   TokenIcon,
   ActionButtons,
-  SliderCard,
 } from '@components/app';
-import { SpinnerLoading, Text, Button } from '@components/common';
+import { SpinnerLoading, Text } from '@components/common';
 import { toBN, halfWidthCss, humanize, normalizeAmount } from '@utils';
 import { getConfig } from '@config';
 import { getConstants } from '@config/constants';
 import { device } from '@themes/default';
-import { AmsterdamLogo } from '@assets/images';
 
 const StyledViewContainer = styled(ViewContainer)`
   display: grid;
   grid-template-columns: 1fr 1fr;
   grid-auto-rows: min-content;
 `;
 
-const AmsterdamButton = styled(Button)`
-  background: #0657f9;
-  color: #fff;
-`;
-
-const StyledSliderCard = styled(SliderCard)`
+const AmsterdamCard = styled.div`
   grid-column: 1 / 3;
 `;
```

Comment:
Creating a styled.div to fix the whole width

Reply:
Removed the banner tho